downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -179,9 +179,9 @@ abstract class VersionControlSystem(
     abstract fun getVersion(): String
 
     /**
-     * Return the name of the default branch for the repository at [url], or null if there is no default remote branch.
+     * Return the name of the default branch for the repository at [url].
```

Comment:
Should this now say: "[..], or empty if there is no default remote branch."?

Reply:
No, because all current implementations always return a non-empty default branch as a fallback.

Reply:
> No, because all current implementations always return a non-empty default branch as a fallback.

But isn't this an interface of a plugin, so that potential unknown implementations should be considered as opposed to just the ones existing currently within ORT?

Reply:
While that's technically possible, I'd propose that it's simply not allowed by convention, as I believe there should always by a default branch name that any implementation would be able to fall back to.

Reply:
I've added more docs / text to the commit message.